src/genie/libs/parser/iosxe/show_license.py

```diff
@@ -217,7 +217,14 @@ def cli(self, output=None):
 
         # network-advantage       (C9300-48 Network Advan...)       1 IN USE
         # dna-advantage           (C9300-48 DNA Advantage)          1 IN USE
-        p0 = re.compile(r"^(?P<license>[\w-]+)\s+\((?P<entitlement>.+)\)\s+(?P<count>\d+)\s+(?P<status>.+)$")
+
+        # Example 2
+        # C9300 48P DNA Advantage  (C9300-48 DNA Advantage)          2 AUTHORIZED
+        # C9300 48P Network Adv... (C9300-48 Network Advan...)       2 AUTHORIZED
+        # C9300 24P Network Adv... (C9300-24 Network Advan...)       1 AUTHORIZED
+        # C9300 24P DNA Advantage  (C9300-24 DNA Advantage)          1 AUTHORIZED
+        
```

Comment:
Small couple of changes to this section. 
Drop the Example 2 line. 
Remove the blank line between the examples and the pattern.
Drop the other examples and leave only the top one. They're all very similar so only one example is needed.

Should look like this
```suggestion
        # C9300 48P DNA Advantage  (C9300-48 DNA Advantage)          2 AUTHORIZED
```